pages/[id]/index.js

```diff
@@ -1,16 +1,64 @@
 import initialPictures from "@/store";
 import PostingDetails from "@/components/PostingDetails";
 import { useRouter } from "next/router";
+import CommentForm from "@/components/CommentForm";
+import { useState } from "react";
+import { uid } from "uid";
```

Comment:
What I learned to today as well is that there is native browser support for this:

```javascript
crypto.randomUUID()

```

and you wont need that package anymore.